parlai/scripts/data_stats.py

```diff
@@ -108,6 +108,8 @@ def keep_token(t):
     while not world.epoch_done() and world.total_exs < max_cnt:
         world.parley()
         act = world.get_acts()[opt.get('agent')]
+        if act.is_padding():
```

Comment:
what is this is_padding() field -- do we always have that?
why not just check if the text we get is an empty string?

Reply:
If they are using a `Message` object, instead of `dict`, that method must be there ([link](https://github.com/facebookresearch/ParlAI/blob/fe36d880813a78a0003bd8e03324494caab11ccc/parlai/core/message.py#L49)). 
But maybe skipping the ones that do not have `text` field in them covers a wider number of things that could go wrong as well. I change it to that.

Reply:
Image Teachers often give empty text with an image field. We recently changed messages to explicitly mark when they are padding. It's much cleaner.